.github/workflows/check-state-compatibility.yaml

```diff
@@ -49,39 +52,62 @@ jobs:
 
   check_state_compatibility:
     # DO NOT CHANGE THIS: please read the note above
```

Comment:
"please read the note above"

why not and what is the note above?

Reply:
This is the note above that explains the reason why:
https://github.com/osmosis-labs/osmosis/blob/8a816f4c9c642a0fca493a79d19f7a72e0a03848/.github/workflows/check-state-compatibility.yaml#L21-L29